GitUI/UserControls/FileStatusList.cs

```diff
@@ -781,6 +822,15 @@ public void SetDiff(GitRevision revision)
                 else
                 {
                     GitItemsWithParents dictionary = new Dictionary<string, IList<GitItemStatus>>();
+                    var isMergeCommit = revision.ParentGuids.Count() == 2;
```

Comment:
I think this is an unintended change introduced while solving rebase conflicts. The combined diff was moved to the end of the list.